cli/cmd/login.go

```diff
@@ -200,6 +200,14 @@ func runLogin(ctx context.Context, cmd *cobra.Command) (err error) {
 
 	analytics.TrackLoginSuccess(ctx, invocationUUID.UUID)
 	cmd.Println("CLI successfully authenticated.")
+	cmd.Println(`Logged in. Next, initialize your sync configuration:
```

Comment:
I would prefer it this way:
> CLI successfully authenticated.
> Next, initialize your sync configuration:

"Logged in" seems extra, previous line ("CLI successfully authenticated.") already tells that.